test/integration/camera_tracking.cc

```diff
@@ -176,27 +176,6 @@ TEST(MinimalSceneTest, GZ_UTILS_TEST_ENABLED_ONLY_ON_LINUX(Config))
   EXPECT_GT(10, abs(camera->WorldPose().Pos().Y() - 100));
   EXPECT_GT(10, abs(camera->WorldPose().Pos().Z() - 100));
 
-  // Move target object to new position
-  trackedVis->SetWorldPose({130, 130, 130, 0, 0, 0});
-
-  // Follow
-  result = false;
-  executed = node.Request("/gui/follow", req, timeout, rep, result);
-  EXPECT_TRUE(executed);
-  EXPECT_TRUE(result);
-  EXPECT_TRUE(rep.data());
-
-  msgs::Vector3d reqOffset;
-  reqOffset.set_x(1.0);
-  reqOffset.set_y(1.0);
-  reqOffset.set_z(1.0);
-  result = false;
-  executed = node.Request("/gui/follow/offset", reqOffset, timeout, rep,
-      result);
-  EXPECT_TRUE(executed);
-  EXPECT_TRUE(result);
-  EXPECT_TRUE(rep.data());
```

Comment:
Instead of removing, we whould update the test to use `/gui/track`? I believe the test is failing because we aren't requesting the camera to track the object.

Reply:
Thanks for the suggestion, fixed in 5604930.